packages/vuetify/src/labs/VCalendar/VCalendar.tsx

```diff
@@ -146,41 +156,76 @@ export const VCalendar = genericComponent<VCalendarSlots>()({
                   .map((week, wi) => (
                     [
                       !props.hideWeekNumber ? <div class="v-calendar-month__weeknumber">{ weekNumbers.value[wi] }</div> : '',
-                      week.map(day => (
-                        <VCalendarMonthDay
-                          color={ adapter.isSameDay(new Date(), day.date) ? 'primary' : undefined }
-                          day={ day }
-                          title={ day ? adapter.format(day.date, 'dayOfMonth') : 'NaN' }
-                          events={ props.events?.filter(e => adapter.isSameDay(day.date, e.start) || adapter.isSameDay(day.date, e.end)) }
-                        ></VCalendarMonthDay>
-                      )),
+                      week.map(day =>
+                        slots.day?.({
+                          day,
+                          events: props.events?.filter(e => adapter.isSameDay(day.date, e.start) || adapter.isSameDay(day.date, e.end)),
+                        }) ?? (
+                          <VCalendarMonthDay
+                            color={ adapter.isSameDay(new Date(), day.date) ? 'primary' : undefined }
+                            day={ day }
+                            title={ day ? adapter.format(day.date, 'dayOfMonth') : 'NaN' }
+                            events={ props.events?.filter(e => adapter.isSameDay(day.date, e.start) || adapter.isSameDay(day.date, e.end)) }
+                            { ...getPrefixedEventHandlers(attrs, ':day', () => ({
```

Comment:
question: I'm assuming the dynamic nature of this means that the events can't be documented in/through code? @KaelWD

Reply:
We could have a `genPrefixedEventProps<T>(':day')` too but it would have to add every single dom event as a prop which would be a huge mess

Reply:
yeah.. might need that override again